pom.xml

```diff
@@ -44,8 +44,9 @@
         <air.maven.version>3.3.9</air.maven.version>
 
         <dep.antlr.version>4.7.1</dep.antlr.version>
-        <dep.airlift.version>0.183</dep.airlift.version>
-        <dep.packaging.version>${dep.airlift.version}</dep.packaging.version>
+        <dep.airlift.version>0.183.3</dep.airlift.version>
+        <!-- <dep.packaging.version>${dep.airlift.version}</dep.packaging.version> -->
+        <dep.packaging.version>0.183</dep.packaging.version>
```

Comment:
Can't we use 0.183.3 here?

Reply:
I changed it to fix the following error.
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-assembly-plugin:2.4:single (package) on project presto-thrift-testing-server: Execution package of goal org.apache.maven.plugins:maven-assembly-plugin:2.4:single failed: Plugin org.apache.maven.plugins:maven-assembly-plugin:2.4 or one of its dependencies could not be resolved: Could not find artifact io.airlift:packaging:jar:0.183.3 in central (https://repo.maven.apache.org/maven2) -> [Help 1]
```

Reply:
I needed to add `pluginRepositories` tag.